examples/make_flow/__init__.py

```diff
@@ -1,4 +1,4 @@
 from examples.make_flow.make_realnvp_normflow import make_wrapped_normflowdist, make_normflow_model, make_normflow_snf_model
 from examples.make_flow.make_masked_affine_nflows import make_wrapped_nflows_dist
 
-from examples.make_flow.make_aldp_model import make_aldp_model
\ No newline at end of file
+# from examples.make_flow.make_aldp_model import make_aldp_model
```

Comment:
Adding the aldp stuff to the `__init__.py` files means that we have to install quite a few extra dependencies. I think we should make these denendencies optional so I commented this out. What do you think?

Reply:
I see that it is annoying for you to install them now when they are not needed, so this is fine for now.